src/parser/warrior/protection/modules/spells/ShieldBlock.js

```diff
@@ -1,102 +1,214 @@
 import React from 'react';
-import { formatPercentage, formatThousands } from 'common/format';
-import SpellIcon from 'common/SpellIcon';
 import SpellLink from 'common/SpellLink';
-import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
 import Analyzer from 'parser/core/Analyzer';
 import SPELLS from 'common/SPELLS';
+import Statistic from 'interface/statistics/Statistic';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText/index';
 
 const debug = false;
 
 class ShieldBlock extends Analyzer {
-  lastShieldBlockBuffApplied = 0;
+
+  shieldBlocksO;
```

Comment:
Again, please keep variables descriptive. What does O, D and Ov stand for here? It's not obvious out of immediate context.